Src/Couchbase.Linq/Extensions/BucketExtensions.cs

```diff
@@ -8,7 +9,8 @@ public static class BucketExtensions
     {
         public static IQueryable<T> Queryable<T>(this IBucket bucket)
         {
-            return EntityFilterManager.ApplyFilters(new BucketQueryable<T>(bucket));
+            //TODO refactor so ClientConfiguration is injectable
+            return EntityFilterManager.ApplyFilters(new BucketQueryable<T>(bucket, new ClientConfiguration()));
```

Comment:
I see, I think maybe the solution is to make BucketExtensions.Queryable internal (just for ease of testing), since the API we want public  is IDbContext. Doesn't make a lot of sense to have two different API's.

I agree BucketContext...I'll follow up with a commit that does this.

-Jeff